tests/yang/YANGBearTest.py

```diff
@@ -0,0 +1,24 @@
+import requests
+
+from bears.yang.YANGBear import YANGBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+
+VALID = requests.get(
```

Comment:
Please store this in our repo, or create simple tests if that file cant be licenced as MIT.